src/client/src/ui/analytics/components/Analytics.tsx

```diff
@@ -96,14 +82,14 @@ export default class Analytics extends React.Component<Props> {
 
 function lastPositionWithDirection(lastPos: number) {
   let formattedLastPos = numeral(lastPos).format()
-  let color = ''
+  const color = ''
```

Comment:
Only use const if the value is constant. Because we change the value later, this must be let.

Reply:
Oh I didn't mean to commit that